dangerfile.js

```diff
@@ -18,6 +18,6 @@ const {danger, warn, fail} = require('danger')
 
 // Danger.js documentation: http://danger.systems/js/
 
-if (danger.github.pr.body.length < 10) {
-  warn('This PR does not include a description.');
+if (!danger.github.pr.title.startsWith("[")) {
```

Comment:
What do you we do for multiple components? Previously, I might omit the component in the summary and just tag the components with labels.

```
[ActivityIndicator, BottomNavigation, Buttons, ButtonBar,  Slider, Tabs, Text Fields] Update BUILD file `includes` paths
```

Reply:
If something is affecting multiple components and the change can't be broken up into multiple PRs, then the common trait of the change could be used in the PR title. E.g. `[documentation]`.

Reply:
Ok. I wasn't sure how to interpret 'component' below - I thought it had to map to a specific component.

Reply:
Ah that's fair. I'll flesh out the description in a follow-up PR.

Reply:
https://github.com/material-components/material-components-ios/pull/3757